### PR TITLE
Add detection for SSR / responses in result HTML

### PR DESCRIPTION
**Pull request by dan-lee**

### CODE REVIEW

**Review by mxstbr — COMMENTED** [review]
The logic looks great! My only additional feedback is that the false positive detection that's currently in the `webRequest.onCompleted` has to run for this as well. Maybe there's a listener that runs after `tabs.onUpdated` that we could use instead of the onCompleted one?

**Comment by dan-lee** [comment]
Hmm, I am not sure I can quite follow :) For now, I've just added the false positive check into the same listener. See a17a6dc

File: src/app/background.ts [hunk 1]
```diff
@@ -21,6 +21,36 @@ const handlers = {
   getResult: (message) => tabs.get(message.tabId),
 };
 
+chrome.tabs.onUpdated.addListener((tabId, changeInfo, tab) => {
+  if (changeInfo.status === "complete" && tab?.status === "complete" && tab.url !== undefined) {
+    chrome.tabs.executeScript(
+      tabId,
+      { code: "document.documentElement.outerHTML.includes('__typename')" },
```
Comment (dan-lee) [inline src/app/background.ts:28]:
```suggestion
      { code: "[...document.querySelectorAll('script')].some((script) => script.textContent.includes('__typename'))" },
```

What if we checked inside `script` tags only? I am not sure where else SSR content might be stored. I figured if a blog post or tutorial mentioned `__typename`, that might be too broad of a heuristic to just check if the whole HTML contained that string

Comment (JoviDeCroock) [reply src/app/background.ts:28]: I think checking within script is a good call

**Review by JoviDeCroock — APPROVED** [review]

**Pull request merged**